engine/verification/utils/fixtures.go

```diff
@@ -139,7 +139,10 @@ func CompleteExecutionResultFixture(t *testing.T, chunkCount int, chain flow.Cha
 		Guarantees: guarantees,
 	}
 	header := unittest.BlockHeaderFixture()
-	header.Height = 0
+	// This is 43 due to the fact that the default mocked sealed height is 42,
```

Comment:
Is that `42` value available as a constant? 

Would be nice to have `header.Height = mockedSealedHeight + 1` or something